examples/control/laner.py

```diff
@@ -37,15 +37,15 @@ def main(scenarios, headless, num_episodes, max_episode_steps=None):
     env = gym.make(
         "smarts.env:hiway-v0",
         scenarios=scenarios,
-        agent_specs=agent_specs,
+        agent_specs=agent_interface,
```

Comment:
This will cause problems.
```suggestion
        agent_specs=agent_specs,
```